modules/dnn/test/test_int.cpp

```diff
@@ -21,6 +27,44 @@ int64_t getValueAt(const Mat &m, const int *indices)
     return -1;
 }
 
+int64_t getValueAt(const Mat &m, int index)
+{
+    if (m.type() == CV_Bool)
+        return m.ptr<bool>()[index];
+    else if (m.type() == CV_8U)
+        return m.ptr<uint8_t>()[index];
+    else if (m.type() == CV_8S)
+        return m.ptr<int8_t>()[index];
+    else if (m.type() == CV_32S)
+        return m.ptr<int32_t>()[index];
+    else if (m.type() == CV_64S)
+        return m.ptr<int64_t>()[index];
+    else
+        CV_Error(Error::BadDepth, "Unsupported type");
+    return -1;
+}
+
+void fillRandom(Mat& m, int matType, Backend backend)
+{
+    if (matType == CV_64S && backend == DNN_BACKEND_INFERENCE_ENGINE_NGRAPH)
+        cv::randu(m, 1000000000, 1000000100); // Looks like OpenVINO uses int32 internal values for int64 operations
+    else if (matType == CV_64S)
+        cv::randu(m, 1000000000000000ll, 1000000000000100ll);
+    else if (matType == CV_32S)
+        cv::randu(m, 1000000000, 1000000100);
+    else if (matType == CV_8S)
+        cv::randu(m, -50, 50);
+    else if (matType == CV_8U)
+        cv::randu(m, 0, 100);
+    else if (matType == CV_Bool)
+    {
+        for (int i = 0; i < m.total(); ++i)
+            m.ptr<bool>()[i] = rand() % 2;
```

Comment:
It's bad idea. OpenCV TS does not control seed for std::rand. You'll get different sequence subset of tests is requested by user, or test order is changed by some reason. cv::RNG::fill may be used for all branches: https://docs.opencv.org/5.x/d1/dd6/classcv_1_1RNG.html#ad26f2b09d9868cf108e84c9814aa682d